src/lib/browse/Filters.svelte

```diff
@@ -28,6 +29,8 @@
   let authorities: [string, string][] = [];
   let filterAuthority = "";
   let fundingProgrammes: [string, string][] = [];
+  let atfFundingProgrammes: string[] =
```

Comment:
This includes planned pipeline... it's everything except "other." How about `knownFundingProgrammes` and making it a Set, for slightly faster checking later?